scripts/build_review_app.sh

```diff
@@ -73,9 +73,16 @@ hokusai review_app env copy $NAME --configmap nginx-config --verbose
 # authentication requests won't work!
 hokusai review_app env set $NAME \
   APP_URL="https://$NAME.artsy.net" \
+  APPLICATION_NAME="$NAME" \
```

Comment:
This makes it so that we _don't_ see the banner? Not sure if that's we want...I at least find it useful to know that I'm a review app...

I would recommend that we document how to make the banner go away, but have it appear by default after a review app is built via this script.

Reply:
I think it's safe to not have the banner. When we're constantly in staging it's an issue, but for a review app it's different. The banner gets in the way, pushes stuff down.